testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
+
+## Thy tests should be obvious
+If the test behaviour isn't obvious by looking at it, it's no good. If I have to scroll for days to read a test, it's no good.
+
+## Thy tests should be loosely coupled with thy code
```

Comment:
I stand by the clarification of the ambiguity on the line below - `If I make a single code change and many tests fail, it's no good.`. If you can provide a clear definition that doesn't use the words `coupled` or `interface` that forms a better definition, I'm open to hearing it :+1: